src/Views/Calendar/MonthView.js

```diff
@@ -723,18 +737,32 @@ const __class = declare('crm.Views.Calendar.MonthView', [List, _LegacySDataListM
 
   renderCalendar: function renderCalendar() {
     const calHTML = [];
-    const startingDay = this.getFirstDayOfCurrentMonth().day();
-    const dayDate = this.currentDate.clone().startOf('month');
+    const startingDay = this.getFirstDayOfCurrentMonth()
+      .day();
+    const dayDate = this.currentDate.clone()
+      .startOf('month');
     const monthLength = this.currentDate.daysInMonth();
     const weekEnds = [0, 6];
     let weekendClass = '';
     let day = 1;
+    let weekdaysText = this.weekdaysShortText;
```

Comment:
Must have missed this little piece when restructuring to the resource.value format for the strings. This was the method used to get around the object creation not working for l20n and thus should be removed since the refactoring. The this.weekdaysShortText will always exist now (or should) and therefore the weekdaysText variable is redundant, and setting the array is certainly redundant.